shadows/framework/src/main/java/org/robolectric/shadows/RangingResultBuilder.java

```diff
@@ -88,6 +89,12 @@ public RangingResultBuilder setIs80211mcMeasurement(boolean is80211mcMeasurement
     return this;
   }
 
+  // This was added in Android V.
+  public RangingResultBuilder setIs80211azNtbMeasurement(boolean is80211azNtbMeasurement) {
```

Comment:
Did you mean `set80211azNtbMeasurement`?
https://developer.android.com/reference/kotlin/android/net/wifi/rtt/RangingResult.Builder?hl=en#set80211azntbmeasurement

And shouldn't the method be annotated with `@Implementation(minSdk = V)`?

Reply:
Hey @MGaetan89 this is a Builder, not a Shadow, so you don't need the `@Implementation` methods